hooks/build

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env sh
+
+# this script is used to build a Radicale Docker image locally
+# as well as on the Docker Hub.
```

Comment:
Dockerhub doesn't run uncontained shellscripts.

Reply:
not sure what's your point here.

Reply:
DockerHub just asks you, where your Dockerfile is located and directly builds the Docker image. You can't run this script on DockerHub. This comment is either just a minor mistake or - worse and I hope not - yields basic issues in your understanding of DockerHub, which will actually doom most concepts of this PR.

Reply:
nah, it dooms only the metadata as `label-schema.org` proposes. you're right, i had the whole time the misconception that how builds are performed [on the Docker Cloud](https://docs.docker.com/docker-cloud/builds/automated-build/#configure-automated-build-settings) is identical as on the Hub. however, i couldn't checkout whether this approach actually works when a build is configured on the Cloud frontend as i can't connect a repo there to a repo here (maybe one must pay for this).

so, i could just remove that metadata labels.

Reply:
oh, i just rechecked my local copies of an image that was [built](https://hub.docker.com/r/funkyfuture/deck-chores/builds/bhcoepocy5o7frwaswa8iee/) on the Docker Hub with the hooks a month ago, and it actually has the expected metadata. though the build log on the Hub look different.

from the discussion on [this issue](https://github.com/docker/hub-feedback/issues/508) i just realised that the hooks dir needs to be moved to the `docker` dir.